insights/parsers/ovs_bridge_mac_table_entry_count.py

```diff
@@ -0,0 +1,67 @@
+"""
+OVSappctlFdbShowBridgeCount - command ``/usr/bin/ovs-appctl fdb/show [bridge-name]``
+====================================================================================
+
+This module provides class ``OVSappctlFdbShowBridgeCount`` to parse the
+output of command ``/usr/bin/ovs-appctl fdb/show [bridge-name]`` and returns
+the MAC entry count.
+
+Sample command output::
+
+    port VLAN  MAC Age
+    6       1 MAC1 118
+    3       0 MAC2 24
+
+The content collected by insights-client::
+
+    {"br-int": 6, "br-int1": 21, "br-tun": 1, "br0": 0}
+
+Examples:
+    >>> data["br-int"]
+    6
+    >>> data.get("br0")
+    0
+    >>> "br-int1" in data
+    True
+"""
+
+import json
+from insights import CommandParser, LegacyItemAccess, parser
+from insights.parsers import ParseException, SkipException
+from insights.specs import Specs
+
+
+@parser(Specs.ovs_bridge_mac_table_entry_count)
+class OVSappctlFdbShowBridgeCount(CommandParser, LegacyItemAccess):
```

Comment:
Ideally a parser shouldn't be so specific to a particular use case. Why doesn't this parser represent the table of information provided by the command?

Reply:
@csams -  Since the command output contains MAC addresses which is a sensitive data, it was decided to get the MAC count for a bridge in the client slide only as the plugin needed only the MAC count per bridge. So, the parser is only parsing the bridge name and MAC count.
If it is allowed to collect MAC address, I'll enhance the parser as per suggestions.